native/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -222,6 +260,90 @@ public static SyntaxTree generateClientSyntaxTree(Entity entity) {
                         ))));
             }
         }
+        if (entity.getRelations().size() != 0) {
+            for (Relation relation : entity.getRelations()) {
+                if (relation.getRelatedFields().size() == 0) {
+                    continue;
+                }
+                StringBuilder refFieldsString = new StringBuilder();
+                StringBuilder joinColumnsString = new StringBuilder();
+
+                for (int i = 0; i < relation.getKeyColumns().size(); i++) {
+                    if (refFieldsString.length() != 0) {
+                        refFieldsString.append(", ");
+                        joinColumnsString.append(", ");
+                    }
+                    refFieldsString.append(relation.getReferences().get(i));
+                    joinColumnsString.append(relation.getKeyColumns().get(i));
+                }
+                if (!joinSubFields.isEmpty()) {
+                    joinSubFields.add(NodeFactory.createBasicLiteralNode(SyntaxKind.STRING_LITERAL,
+                            AbstractNodeFactory.createLiteralValueToken(SyntaxKind.STRING_LITERAL, ", "
+                                            + System.lineSeparator(), NodeFactory.createEmptyMinutiaeList(),
+                                    NodeFactory.createEmptyMinutiaeList())));
+                }
+                joinSubFields.add(NodeFactory.createSpecificFieldNode(null,
+                        AbstractNodeFactory.createIdentifierToken(relation.getRelatedInstance()),
+                        SyntaxTreeConstants.SYNTAX_TREE_COLON,
+                        NodeParser.parseExpression(String.format(
+                                BalFileConstants.FIELD_FORMAT_JOIN_FIELD, "entities:" + relation.getRelatedType(),
```

Comment:
Let's move `"entities:"` to constants.

Reply:
updated in [2bf9a5f](https://github.com/ballerina-platform/persist-tools/pull/23/commits/2bf9a5f4d1f39a2145630441c20b06f8d3c16ef9)